dtmi/azureiot/phone_as_a_device/sensors/basesensor-2.json

```diff
@@ -0,0 +1,59 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:azureiot:Phone_As_A_Device:Sensors:baseSensor;2",
```

Comment:
if not using extends, we could remove this base interface

Reply:
As services make fuller use of extends, we can update the application to use the updated templates and these base interfaces. It doesn't make sense to discard work that has been done just because services haven't yet implemented the standards fully.

Reply:
the DMR makes the promise that these interfaces can be used with our services. Without Central and/or IoT Explorer support we need to be very prescriptive. Taking this discussion offline.